src/components/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -318,6 +378,19 @@ export const HomeScreen: React.FC<HomeScreenProps> = ({ navigation }) => {
         network={network}
         onTokenPress={handleTokenPress}
       />
+
+      <WelcomeBannerBottomSheet
+        modalRef={welcomeBannerBottomSheetModalRef}
+        onAddXLM={() => {
+          navigation.navigate(ROOT_NAVIGATOR_ROUTES.BUY_XLM_STACK, {
+            screen: BUY_XLM_ROUTES.BUY_XLM_SCREEN,
+            params: { isUnfunded: hasZeroBalance },
+          });
```

Comment:
this code is duplicated from the "home.buy" button. Could you please create a function for this so we could re-use on both places?